packages/esm-service-queues-app/src/patient-search/patient-search.workspace.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { SearchTypes } from '../types';
+import PatientScheduledVisits from './patient-scheduled-visits.component';
+import VisitForm from './visit-form/visit-form.component';
+import {
+  type DefaultWorkspaceProps,
+  ExtensionSlot,
+  usePatient,
+  useVisit,
+  PatientBannerPatientInfo,
+  PatientPhoto,
+  PatientBannerToggleContactDetailsButton,
+  PatientBannerContactDetails,
+} from '@openmrs/esm-framework';
+import ExistingVisitFormComponent from './visit-form/existing-visit-form.component';
+import styles from './patient-search.scss';
+
+interface PatientSearchProps extends DefaultWorkspaceProps {
+  viewState: {
+    selectedPatientUuid?: string;
+  };
+}
+
+const PatientSearch: React.FC<PatientSearchProps> = ({ closeWorkspace, viewState }) => {
+  const { selectedPatientUuid } = viewState;
+  const { patient } = usePatient(selectedPatientUuid);
+  const { activeVisit } = useVisit(selectedPatientUuid);
+  const [searchType, setSearchType] = useState<SearchTypes>(SearchTypes.SCHEDULED_VISITS);
+  const [newVisitMode, setNewVisitMode] = useState<boolean>(false);
+  const [showContactDetails, setContactDetails] = useState(false);
+
+  const toggleSearchType = (searchType: SearchTypes, mode: boolean = false) => {
+    setSearchType(searchType);
+    setNewVisitMode(mode);
+  };
+
+  const patientName = `${patient?.name?.[0]?.given?.join(' ')} ${patient?.name?.[0].family}`;
```

Comment:
We probably want to leverage the `displayName` utility function from Core similar to how it's done [here](https://github.com/openmrs/openmrs-esm-patient-management/pull/1141/files#diff-6d85d66cd23e772889646c09adcf6a9e1dde1211a52cf32f29bd00ec61205e08R113).

Reply:
Love that we have this now